### PR TITLE
Feature/milestone 3

### PR DESCRIPTION
**Pull request by martin-cervantes**
:heavy_check_mark: Add style to App. Display, ButtonPanel, Groups, and Buttons according to project specifications
:heavy_check_mark: Pass color and wide props to Buttons

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Button.propTypes = {
+  name: PropTypes.string,
+  color: PropTypes.string,
+  wide: PropTypes.bool,
+};
+
+Button.defaultProps = {
+  name: 'Button',
+  color: '',
```
Comment (wathigo) [inline src/components/Button.js]: **REQUIRED**
* The button default color should be orange unless changed by a color prop([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/colorProps.png) from

**Review by wathigo — CHANGES_REQUESTED** [review]
### Changes Required :heavy_multiplication_x: 
Great job on this milestone.
Let's make some changes before we move on.

**REQUIRED**
* The project has a descriptive README.md ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_readme.png) from Pathwright).

**After making the changes, submit another request for review.**

**Simon Wathigo.**

**Comment by martin-cervantes1** [comment]
:heavy_check_mark: Add readme file :memo: 
:heavy_check_mark: Changed button default color to orange :orange_circle:

**Review by mikenath223 — APPROVED** [review]
### CONGRATULATIONS  👏👏👏

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**